04.ls/ls.rb

```diff
@@ -5,14 +5,26 @@
 PADDING = 2
 
 def main
-  path = ARGV[0] || '.'
-  FileTest.directory?(path) or return
-  file_names = Dir.children(path).sort
-  display_file_names(file_names)
+  path = '.'
+  option_settings = { is_option_a_enabled: false }
```

Comment:
```suggestion
  options = { a: false }
```

ちょっと命名が冗長なので、これぐらいで十分かもしれません。